pmoves/services/agentgym-rl-coordinator/coordinator/publisher.py

```
@@ -0,0 +1,373 @@
+        if resp.status_code != 200:
+            raise RuntimeError(f"Failed to get trajectory: {resp.text}")
+
+        trajectories = resp.json()
+        if not trajectories:
+            raise ValueError(f"Trajectory not found: {trajectory_id}")
+
+        trajectory = trajectories[0]
+
+        # Generate output path
+        if not output_path:
+            output_path = f"/tmp/trajectory_{trajectory_id}.json"
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/trajectory_"

(S108)